File: modules/stylus-input/ios/StylusInputModule.swift
import ExpoModulesCore

public class StylusInputModule: Module {
  public func definition() -> ModuleDefinition {
    Name("StylusInput")

    View(StylusInputView.self) {
      Prop("enabled") { (view: StylusInputView, enabled: Bool) in
        view.isInputEnabled = enabled
      }
      Events("onStylusBatch")
    }
  }
}

File: modules/stylus-input/android/src/main/java/expo/modules/stylusinput/StylusInputModule.kt
package expo.modules.stylusinput

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class StylusInputModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("StylusInput")

    View(StylusInputView::class) {
      Prop("enabled") { view: StylusInputView, enabled: Boolean ->
        view.inputEnabled = enabled
      }
      Events("onStylusBatch")
    }
  }
}
